CONTRIBUTING.md

```diff
@@ -64,15 +71,23 @@ If you don't have permissions to the main repository, then you can ignore this s
 
 * Before a pull request can be landed it should:
   * Pass all automated tests
-  * Have been reviewed
-  * Have an appropriate commit structure
+  * Have been reviewed and been given a +r by the reviewer
+  * Have an appropriate commit structure (It should be one commit, or a couple of commits, if appropriate)
   * Each commit should have an appropriate message, e.g.
     * `Bug 123456 – Loop's RoomList view requires user profile data passed in when
 it doesn't need to. r=Standard8`
-    * If the PR is being merged with a merge commit due to limitations of branches,
-      then the merge commit should have the `r=Standard8` included in it.
-* If possible, a pull request should be landed without a merge commit. To do this:
+    * If the PR is being merged with Merge Pull Request due to limitations of branches,
+      then the Merge Pull Request comment should include `r=<reviewerName>`
+      or in the case of this example `r=Standard8`.
+* If possible, a pull request should be landed without a Merge Pull Request.
+  To do this:
   * the branch should be rebased on top of latest master
+    * `git checkout master`
+    * `git pull mozilla master`
+    * `git checkout <branch>`
+    * `git rebase -i master`
```

Comment:
@Standard8, @edilee doesn't this want to be something like:

```shell
git push origin functional-test-fix :master
```

Which is less dangerous since it avoids the -f ?